test/e2e/nvidia-container-toolkit_test.go

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+// Actual test suite
+var _ = Describe("NVIDIA Container Toolkit", func() {
+	JustBeforeEach(func(ctx context.Context) {
+		// Placeholder for any setup that needs to be done before each test
+	})
+
+	AfterEach(func(ctx context.Context) {
+		// Placeholder for any cleanup that needs to be done after each test
+	})
+
+	// GPUs are accessible in a container: Running nvidia-smi -L inside the
+	// container shows the same output inside the container as outside the
+	// container. This means that the following commands must all produce
+	// the same output
+	When("Running nvidia-smi -L", Ordered, func() {
```

Comment:
Is `Ordered` required here? Is there a way that the different mechanisms to select devices are treated differently?

Reply:
from documentation: `Ordered is a decorator that allows you to mark a container as ordered. Specs in the container will always run in the order they appear. They will never be randomized and they will never run in parallel with one another, though they may run in parallel with other specs.` 
So is not required, we can remove it if we want.

Reply:
There are no specs in this container though, or am I misunderstanding what a "Spec" is? Can we rewrite the separate tests as Specs?

Reply:
you mean an `It()` for each run?

Reply:
I have updated the PR description to reflect new changes related to this thread

Reply:
Let's also update the title to make it more specific.